src/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Todo-list-app</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css" integrity="sha512-SzlrxWUlpfuzQ+pcUCosxcglQRNAq/DZjVsC0lE40xsADsfeQoEypE+enwcOiGjk/bSuGGKHEyjSoQ1zVisanQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+  <div class="todo-list-cont">
+    <div class="todays-todo">
+
+    </div>
+    <div class="add-todo">
+      
+    </div>
+    
+    <ul class="todo-list">
+
+    </ul>
+  
+    <div class="clearAll">
+      <button class="clearAllBtn">Clear all completed</button>
+    </div>
+  </div>
```

Comment:
In the body,  you can wrap your content in the `main` tag instead of `div` for semantics and to make the content inside the main to be unique in your webpage.

Reply:
Thank you for your constructive comment. I will do it.